vortex-array/src/stats/statsset.rs

```diff
@@ -55,9 +55,11 @@ impl StatsSet {
 
     pub fn constant(scalar: Scalar, length: usize) -> Self {
         let mut stats = Self::default();
-        stats.set(Stat::IsConstant, true);
-        stats.set(Stat::IsSorted, true);
-        stats.set(Stat::IsStrictSorted, length <= 1);
+        if length > 0 {
```

Comment:
I think it's weird to flag this just for constant, e.g. empty null array will have all of these set to true